nservicebus/messaging/headers.md

```diff
@@ -89,6 +89,16 @@ Identifier of the conversation that this message is part of. It enables the trac
 
 The first message that is sent in a new flow is automatically assigned a unique `Conversation Id` that is then propagated to all the messages that are subsequently sent, thus forming a _conversation_. Each message that is sent within a conversation also has a `RelatedTo` value that identifies the originating message that caused it to be sent. 
 
+There are scenarios where you want to manually set the Conversation Id, for example, a CancelOrder needs to become part of an existing order conversation. This can be done by overriding the header with a custom value:
+
+snippet: override-conversation-id
+
+It is not possible to override an existing Conversation Id, trying to do this results in the following error:
```

Comment:
Should this be a `NOTE:` ?

Perhaps also add that messages emitted from handlers will get the conversation id of the incoming message so its not really possible to override conversation ids from message handlers?

Reply:
@andreasohlund The exception that follows it is part of it. I don't think that can be combined.

Reply:
ok

Reply:
@andreasohlund the section already has the following: